### ISSUE TITLE
Do the SDK really need to set an AppTheme?

### ISSUE THREAD

**Issue by rocboronat** [citrix/citrix-mam-sdks#42]
Our project, as a lot of projects out there, specifies an app theme.

It seems that the SDK also specifies an app theme in its AndroidManifest, so it conflicts with the project's one.

The question: is it really needed?

This is the error message it launches:
```
Manifest merger failed : Attribute application@theme value=(@style/FewlapsTheme) from AndroidManifest.xml:37:9-45
	is also present at [com.citrix.android:authmanagerlitelib:Release_21.5.0-1] AndroidManifest.xml:15:9-40 value=(@style/AppTheme).
	Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:30:5-309:19 to override.
```

**Comment by terry-citrix** [comment]
Hi @rocboronat , we cover this issue a bit in our Troubleshooting section.  See https://developer.cloud.com/citrixworkspace/mobile-application-integration/android-native/docs/troubleshooting#how-to-resolve-a-gradle-build-error-androidmanifestxml-error-attribute-applicationtheme-is-also-present-at-comcitrixandroidauthmanagerlitelib-

We probably should not define a theme in that library, and that's something that we'll look into.  In the meantime you can override this theme with your own.  The suggestion is to add 'tools:replace="android:theme"' to the <application> element in the AndroidManifest.xml file.

**Comment by rocboronat** [comment]
Yes, obviously I did it, but please, fix it for good. Closing this issue is unexpected if you intend to keep the project sane.

**Comment by terry-citrix** [comment]
Ah, I see.  You were asking for a code change, rather than asking a technical question.  I've re-opened the issue so that we can track it publicly as an enhancement request.

**Comment by rocboronat** [comment]
Yap. Thanks for reopening the issue 😊
